utils_cv/detection/plot.py

```diff
@@ -0,0 +1,279 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
General though about plotting, and don't know the answer: maybe there is a common library we could use for plotting, etc, rather than having to implement it ourselves. Something to keep in mind going forward..